x-pack/plugin/src/test/resources/rest-api-spec/test/privileges/11_builtin.yml

```
@@ -15,5 +15,5 @@ setup:
   # This is fragile - it needs to be updated every time we add a new cluster/index privilege
   # I would much prefer we could just check that specific entries are in the array, but we don't have
   # an assertion for that
-  - length: { "cluster" : 33 }
+  - length: { "cluster" : 34 }
   - length: { "index" : 17 }
```

You're adding a new _index_ privilege not a _cluster_ one, so I believe this should instead be:
```suggestion
   - length: { "index" : 18 }
```